Pocket/pocket-ios
# feat(Braze) Enable Migration Event IN-1213

## Summary
Add new Braze event 'SIGNED_IN_USER_UPGRADE_DID_START' when user begins migration (presumably from 7 to 8)


## Implementation Details
Added `signedInUserDidBeginMigration()` func to PocketBraze. When a migration is to be performed a closure should be passed into the perform function that contains a call to the above function.

## Test Steps
Set up a user with a v7 app and account
Log into v8 and check Braze for the event.

## PR Checklist:
